### PR TITLE
feat: add user verification after signup

### PR DESCRIPTION
**Pull request by Vansh353**

### CODE REVIEW

File: controllers/user_controller.py [hunk 1]
```diff
@@ -1,31 +1,45 @@
...
-    
     user = User(
         name=user_dto.name,
         email=user_dto.email,
+        
         password=hashed_password,
-    
     )
-
+    email_list.append(user_dto.email)
```
Comment (niyati-doshi20) [inline controllers/user_controller.py]: Find some other solution to get email in list. As we have only one email.

File: helpers/email_helper.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <p>Thanks for choosing seabasket, please click on the button below to verify your account</p>
+    <a style="margin-top:1rem; padding:1rem; border-radius:0.5rem; font-size:1rem; text-decoration: none; background:"#027d8; color:white;" href="http://localhost:8000/verification/?token={token}">
+    Verify your email</a>
+
+    <p> Please Kindly ignore this message if you did not sign up for an account.Thanks</p>
+
+    </div>
+    </body>
+    </html>
+    """
```
Comment (niyati-doshi20) [inline helpers/email_helper.py]: Remove template from here and create new file for template and also make email function reuseable

File: helpers/email_helper.py
Comment (niyati-doshi20) [inline helpers/email_helper.py:1]: Why there is 2 functions for send mail?

Comment (Vansh353) [reply helpers/email_helper.py:1]: removed one out of that

Comment (niyati-doshi20) [reply helpers/email_helper.py:1]: Remove duplicate imports and unused imports

**Pull request merged**